src/main/scala/edg_ide/runner/PDFGeneratorUtil.scala

```diff
@@ -48,22 +102,54 @@ object PDFGeneratorUtil{
         }
 
         val cb = writer.getDirectContent
-        val graphics = cb.createGraphics(width, height)
+        val graphics = cb.createGraphics(width, adjustedHeight)
         val painter = new ElkNodePainter(node)
         painter.paintComponent(graphics, Color.white)
         graphics.dispose()
+
+        // Prints out table if there are multiple usages of the same component
+        if(dupSet.size > 1) {
+          val table = new PdfPTable(1)
+
+          val message = new Paragraph(s"${className.toSimpleString} is also used at the following:")
+          val title = new PdfPCell(message)
+          title.setBorder(Rectangle.NO_BORDER)
+          table.addCell(title)
+
+          dupSet.foreach { path =>
+            val parentAnchor = new Anchor
```

Comment:
Can you have this display the full pathname instead of just the parent?

Actually, it might be worth breaking that out into its own function, and having that shared across both the page header as well as the table entries.

Reply:
- are you referring to the part where I made the parent's design path clickable instead of the whole pathname? 
    - I did that initially but I thought it made more sense to only make the parent the hyperlink so that it's obvious to user that the component was used in those respective blocks.

- ok, I will look at breaking out this into its own function

Reply:
Oh, sorry, I mis-read things, it looks like it's already printing the full design path. Though I think it still might make sense to break out the components, for example if it's `usb.cc_pull.cc1`, both the `usb` and `cc_pull` are clickable and go to their respective blocks, instead of both being one link that only goes to `usb.cc_pull`.

And if this is re-used for the page design path, this also serves the function of navigation upwards.